docs/structure-samples/python/python.md

```diff
@@ -0,0 +1,50 @@
+## Python project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a Python standard application.
+
+```bash
+
+~root
+│
+├── .devcontainer.json              
+├── azure.yaml                      
+│
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml           
+│       ├── workflow2.yml           
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep                  
+│   ├── main.parameters.bicep       
+│   ├── abbreviations.json          
+│   └── ...                         
+│
+├── src/                            
+│   ├── projectname/                
+│   │   ├── __init__.py             
+│   │   ├── core/                   
```

Comment:
I don't feel strongly that every project should have core/ models/ and services/. Maybe this specific set of projects would have that? But I don't have those generally.

Reply:
Should I completely remove them or should I add an asterisk for optional and legend?

Reply:
I think just remove them?

Reply:
Removed!